cmd/debugPreprocess.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/StackExchange/dnscontrol/pkg/normalize"
+	"github.com/urfave/cli"
+)
+
+var debugPreprocessCommand = &cli.Command{
+	Name:  "debug-preprocess",
+	Usage: "Run validation and normalization logic, and print resulting json",
```

Comment:
Output intermediate representation (IR) after running validation and normalization logic.